modules/openapi-generator-core/src/main/java/org/openapitools/codegen/config/GeneratorSettings.java

```diff
@@ -38,6 +38,8 @@ public final class GeneratorSettings implements Serializable {
     private String modelPackage;
     private String invokerPackage;
     private String packageName;
+
+    private String resourceFolder;
```

Comment:
Should resoureFolder be removed from this file as it has been moved to SpringCodegen ?

Reply:
I think not, because you have to set it into the Maven Plugin.

It's used here CodegenConfigurator.

```
    public CodegenConfigurator setResourceFolder(String resourceFolder) {
        if (StringUtils.isNotEmpty(resourceFolder)) {
            addAdditionalProperty(SpringCodegen.RESOURCE_FOLDER, resourceFolder);
        }
        generatorSettingsBuilder.withResourceFolder(resourceFolder);
        return this;
    }
```

I thought when i remove this then i cannot override it.

Reply:
Removed

Reply:
So but now it should be :)